### DISCUSSION TITLE
🟢 Harness | Angular Challenges

### DISCUSSION THREAD

**Discussion by giscus** [tomalaforge/angular-challenges#618] [Announcements]
# 🟢 Harness | Angular Challenges

Challenge 23 is about testing with component harnesses

https://angular-challenges.vercel.app/challenges/testing/23-harness/

**Comment by ronnain** [comment]
Hi,
I have an issue when I run this command:
nx test testing-harness

Module <rootDir>/src/subscription-setup.ts in the setupFilesAfterEnv option was not found.

I have already try some solutions here but it is not working:
https://stackoverflow.com/questions/65106848/jest-configuration-setupfilesafterenv-option-was-not-found

Do you have any idea how to fix that ?

**Reply by tomalaforge** [reply]
@ronnain. you are right. All applications were impacted.

I fixed it. If you update your project, the command should work.

**Reply by ronnain** [reply]
Nice, it is working :D
